src/front-end/typescript/lib/pages/landing.tsx

```diff
@@ -50,30 +52,35 @@ const update: Update<State, Msg> = ({ state, msg }) => {
 
 const view: ComponentView<State, Msg> = ({state, dispatch}) => {
   return (
-    <Row>
-      <Col xs='12'>
-        Landing page coming soon.
-      </Col>
-      <Col xs='12' className='pt-7'>
-        <button onClick={() => dispatch(adt('showToast'))}>Show Toast</button>
-      </Col>
-      <Col xs='12' md='8' className='pt-7'>
-        <Accordion
-          toggle={() => dispatch(adt('toggleAccordion'))}
-          color='info'
-          title='Inception'
-          titleClassName='h3'
-          icon='map'
-          iconWidth={2}
-          iconHeight={2}
-          iconClassName='mr-3'
-          chevronWidth={1.5}
-          chevronHeight={1.5}
-          open={state.showAccordion}>
-          Children!
-        </Accordion>
-      </Col>
-    </Row>
+    <Container xs='12'>
+      <Row>
+        <Col xs={{ size: 6, offset: 3 }} className='text-center mt-3'>
+          <h1 style={{
+            lineHeight: '60px'
+          }}>
+            Discover Unique Opportunities to Collaborate with the BC Public Sector.
+          </h1>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs={{ size: 6, offset: 3 }} className='text-center'>
+          The Digital Marketplace is a new platform that will help build an ecosystem of innovation and collaboration between tech entrepreneurs and BC's public sector.
+        </Col>
+      </Row>
+      <Row>
+        <Button className='mx-auto mt-6 mb-6' color='primary'><Icon hover name='search' /><span className='ml-2'>Browse Opportunities</span></Button>
+      </Row>
+      <Row>
+        <Col className='text-center'>
+          <span style={{fontSize: '24px', fontWeight: 'bolder', verticalAlign: 'middle'}}>{formatAmount(1346)}</span>
```

Comment:
Try to use class utilities when possible. In this case:

```tsx
<span className='h4 mb-0'>
...
```

Try to use flexbox over `text-center` and `verticalAlign`.

Also, if you must use `style`, try to specify values in `rem`s. `1rem` is `16px` in this app.